QTeachersApp/View/create.cpp

```diff
@@ -0,0 +1,99 @@
+#include "create.h"
+#include "ui_create.h"
+
+namespace HubertiusNamespace
+{
+    Create::Create(QWidget *parent, QSqlDatabase * database):
+        myTeachersDatabase(database),
+        QWidget(parent),
+        ui(new Ui::Create)
+    {
+        ui->setupUi(this);
+
+    }
+
+    Create::~Create()
+    {
+        delete ui;
+    }
+
+    void Create::on_pushButton_clicked()
+    {
+        // Getting data from lineEdits of ui - START OF CODE
```

Comment:
Zamiast pisać komentarze ze startem/endem (akurat w cpp, zwłaszcza w starszym oprogramowaniu to nie jest niespotykane, but still, postęp technologiczny i dobre nawyki :slightly_smiling_face: ) możesz wydzielać pomniejsze metody, które będą spełniały jedną funkcję.
Na przykład tutaj mógłbyś wywołać sobie metodę prywatną checkDataDuplication, która wywoływałaby w sobie metodę isRowEqual, która porównywałaby Twoje pola z aktualnie wyciągniętym rowem.
Kolejna metoda wywoływana, jeśli metoda od checkDataDuplication nie zwróci informacji że dane są zduplikowane i nie chcemy ich, to np. addNewTeacher().
I tak dalej. To sprawia, że kod sam się dokumentuje, czytając go widzisz po nazwach co się dzieje w danej metodzie i nie musisz do niej zaglądać, a komentarze tego typu są zwykle wówczas zbędne.

Dodatkowo fajnie byłoby zrobić z tych pól jakąś strukturkę np. Teacher. Ułatwiłoby Ci to operowanie na danych i przekazywanie ich między metodami/klasami.